tests/MyGame/Example/MonsterStorage_grpc.go

```diff
@@ -0,0 +1,123 @@
+//Generated by gRPC Go plugin 
+//If you make any local changes, they will be lost
+//source: monster_test
+
+package Example
+
+import "github.com/google/flatbuffers/go"
+
+import (
+  context "golang.org/x/net/context"
+  grpc "google.golang.org/grpc"
+)
+
+// FlatCodec implements gRPC-go Codec
+
+var Codec string = "flatbuffers"
+
+type FlatCodec struct{}
```

Comment:
@JRonak Sure, if this is global and the same for every user, you can put it in a subdirectory of the go lib.